lib/actions/ssh.coffee

```diff
@@ -118,7 +118,8 @@ module.exports =
 				 root@#{options.deviceIp}"
 
 			if not options.host
-			 command += " docker exec -ti #{container} /bin/sh"
+				shellCmd = '/bin/sh -c $"\'if [ -e /bin/bash ]; then exec /bin/bash; else exec /bin/sh; fi\'"'
```

Comment:
Can we use `'''`/`"""` so that we can avoid having to add coffee-script escaping to the bash string, that'll make it a bit easier to read